src/index.js

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
## Optional change
- [ ] can we have a scenario where after creating a project or a todo, the forms get removed on save without the user needing to reload the page? The problem appears to be there when moving from project to project; the user gets forced to reload the page to previously created todos. Do you mind fixing this?
- [ ] Also can we have the error message actually showing on the form fields when the user attempts to submit a half-filled form? Currently, the rules have been enforced but the user cant sees them on the page.